data/summary.go

```diff
@@ -0,0 +1,19 @@
+package data
+
+import (
+    "time"
+    "go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Summary struct {
+    ID                 primitive.ObjectID  `json:"-" bson:"_id,omitempty"`
+	UserID             string              `json:"UserID" bson:"_userId"`
+
+	LastUpdated        time.Time   `json:"lastUpdated" bson:"lastUpdated"`
+	LastData           time.Time   `json:"lastData" bson:"lastData"`
+	LastUpload         time.Time   `json:"lastUpload" bson:"lastUpload"`
+	AverageGlucose     float64     `json:"avgGlucose" bson:"avgGlucose"`
```

Comment:
Consider leveraging the existing structure from `./data/types/blood/blood.go` to include units (mg/dL vs. mmol/L) along with the raw `avgGlucose` value.